lib/index.js

```diff
@@ -93,7 +93,7 @@ function updateDeeply(target, override) {
     }
 
     for (const key in override) {
-        if (override.hasOwnProperty(key)) {
+        if (Object.prototype.hasOwnProperty.call(override, key)) {
```

Comment:
`96:22  error  Do not access Object.prototype method 'hasOwnProperty' from target object  no-prototype-builtins`

This was manually fixed.